.github/tests/test_model_regression_test_read_dataset_branch_tmpl.py

```diff
@@ -0,0 +1,22 @@
+import pathlib
+import subprocess
+import pytest
+
+TEMPLATE_FPATH = ".github/templates/model_regression_test_read_dataset_branch.tmpl"
+REPO_DIR = pathlib.Path("").absolute()
+TEST_DATA_DIR = str(pathlib.Path(__file__).parent / 'test_data')
+DEFAULT_DATASET_BRANCH = "main"
```

Comment:
This variable `DEFAULT_DATASET_BRANCH` seems to be unused. Should it be in `parametrize`?

Reply:
🤦‍♀️ yep, you're very right